CMakeLists.txt

```diff
@@ -34,7 +34,24 @@ add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
 set(BUILD_TESTS off CACHE BOOL "Build test suite" FORCE)
 set(WITH_LUA off CACHE BOOL "Build with lua support" FORCE)
-add_subdirectory(osm2pgsql)
+set(WWW_PUBLIC /var/www/nominatim CACHE PATH "Public www root")
+set(WWW_APP ${CMAKE_INSTALL_PREFIX}/share/nominatim CACHE PATH "Web application root")
+find_program(OSM2PGSQL osm2pgsql DOC "System installed osm2pgsql")
+if (NOT EXISTS ${OSM2PGSQL})
+        add_subdirectory(osm2pgsql)
+	message(STATUS "Using local osm2pgsql")
+	set(OSM2PGSQL_PATH ${CMAKE_INSTALL_PREFIX}/osm2pgsql/osm2pgsql)
+else()
+	set(OSM2PGSQL_PATH ${OSM2PGSQL})
+	message(STATUS "Using osm2pgsql at ${OSM2PGSQL}")
+endif()
+find_program(OSMOSIS_PATH osmosis)
+if (NOT EXISTS "${OSMOSIS_PATH}")
+        message(FATAL_ERROR "Osmosis is required")
```

Comment:
Changed.
If you could open an issue with requirements for installation I'm happy to work on it. Starting a project that will use Nominatim to feed Komoot for Netherlands region and I'd like to start off with upcoming release, rather then previous, so I have some motivation to get it in shape :).